src/SqlServer.Tests/SqlServer2019Tests.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Energinet DataHub A/S
```

Comment:
Remove copyright header

Reply:
Removed as per comment

Reply:
Hi @glucaci. 

Do you have any further comments to the changes?

Br